src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -668,4 +685,31 @@ public static boolean isTest(@NonNull String shipperName) {
         }
         return true;
     }
+
+
+    // todo  arz fixme read from config file
+    private static void enableCORS(String methods, String headers) {
+        Spark.options("/*", (request, response) -> {
+            String accessControlRequestHeaders = request.headers("Access-Control-Request-Headers");
+            if (accessControlRequestHeaders != null) {
+                response.header("Access-Control-Allow-Headers", accessControlRequestHeaders);
+            }
+
+            String accessControlRequestMethod = request.headers("Access-Control-Request-Method");
+            if (accessControlRequestMethod != null) {
+                response.header("Access-Control-Allow-Methods", accessControlRequestMethod);
+            }
+
```

Comment:
I recall @MocanaAtBroad added some magic header to facilitate better caching, something about `Access-Control-Max-Age`. Should we pull that over as well?

Reply:
https://github.com/broadinstitute/ddp-study-server/blob/6579631f8ccdc3cd480e621999a7fc12e94b889e/pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java#L602

Best bang for the buck performance boost. Allows browser to cache OPTION call result instead of preceding every API call with OPTION.